app/poros/user.rb

```diff
@@ -0,0 +1,12 @@
+class User
+  attr_reader :id, :name, :email, :city, :state, :zip, :password
+  def initialize(data)
+    @id = data[:id]
+    @name = data[:name]
+    @email = data[:email]
+    @city = data[:city]
+    @state = data[:state]
+    @zip = data[:zip]
+    @password = data[:password]
```

Comment:
If we're instantiating user objects to display somewhere, why do they need passwords?